client/src/components/Header/styles.tsx

```diff
@@ -29,11 +30,15 @@ export default makeStyles((theme: Theme) => ({
     paddingRight: '8px',
   },
   title: {
-    padding: '0 8px',
+    padding: '0 8px 0 20px',
```

Comment:
I'm messing with the header in my current branch so this might change but fine for now.